Source/ACE/Entity/Creature.cs

```diff
@@ -495,5 +497,162 @@ protected virtual void UpdateVitalInternal(CreatureVital vital, uint newVal)
                 new ActionChain(this, () => VitalTickInternal(vital)).EnqueueChain();
             }
         }
+
+        public override void SerializeIdentifyObjectResponse(BinaryWriter writer)
```

Comment:
This has a signficiant amount of duplicated code with WorldObject's SerializeIdentifyObjectResponse.  Consider calling "base.SerializeIdentifyObjectResponse", or if that is inappropriate, refactoring such that you deduplicate the code between the two calls.